crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -34,6 +34,7 @@ def upgrade
       case @version
       when "1.9"
         knife_files_1_9
+        reset_target_platform_for_nodes
```

Comment:
Well, check the patch :-) The method is a new one, extracted from `knife_files_1_9`.